frontend/sass/_header.scss

```diff
@@ -81,7 +81,7 @@ $logo-height: 29px;
 }
 
 .header .usa-width-two-thirds {
-  display: flex;
+  // display: flex;
```

Comment:
should we just get rid of the entire selector?

Reply:
Yeah I think using the `usa-width` classes OR building our own flex is the way to go. Using both gets some strange results.